org.civicrm.reqangular/src/common/modules/uib-tabset.js

```diff
@@ -0,0 +1,18 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/modules/templates'
+], function (angular) {
+  'use strict';
+
+  return angular.module("common.uibTabset", [])
```

Comment:
this doesn't need to be a module, you should simply doing this in the `.run()` of the already existing `common.directive` module